app/views/static_pages/home.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>StaticPages#home</h1>
-<p>Find me in app/views/static_pages/home.html.erb</p>
+<h1><%= t(".samapp")%></h1>
+<p>
+  <%= raw t(".content") %>
```

Comment:
raw dùng để làm gì

Reply:
Dạ thì trong đó có mấy đường link của href á anh, e thấy khi em không dùng raw thì lúc chạy chương trình nó sẽ in ra nguyên phần href luôn anh, không click để chuyển trang được.
Khi dùng raw thì sẽ hiện đoạn text kèm theo link khi mình click vào nữa á anh.

Reply:
Em nên tìm hiểu và trả lời mục đích chung của phương thức nhen. Tránh việc trả lời theo vế đang thấy thôi, hoặc cái đang làm thôi nhen.
Link tham khảo: https://apidock.com/rails/ActionView/Helpers/RawOutputHelper/raw